rmf_site_editor/src/site/mod.rs

```diff
@@ -321,24 +304,21 @@ impl Plugin for SitePlugin {
                 add_material_for_display_colors,
                 add_physical_lights,
             )
-                .run_if(AppState::in_site_mode())
+                .run_if(AppState::in_displaying_mode())
                 .in_set(SiteUpdateSet::AssignOrphans),
         )
         .add_systems(
             PostUpdate,
             (
                 add_wall_visual,
-                handle_update_fuel_cache_requests,
-                read_update_fuel_cache_results,
-                reload_failed_models_with_new_api_key,
```

Comment:
I just refactored these and the matching resources into a `FuelPlugin` to make it easier to add it to apps.